src/site/content/en/blog/serial/index.md

```diff
@@ -253,6 +253,66 @@ while (true) {
 }
 ```
 
+To specify the buffer into which data is read instead of the stream allocating a buffer for each chunk, get a [ReadableStreamBYOBReader] by calling `port.readable.getReader({ mode: "byob" })` and read data into a new `ArrayBuffer`. Note that the Web Serial API supports "Bring Your Own Buffer" readers in Chrome 106 or later.
```

Comment:
```suggestion
To specify the buffer into which data is read don't allocate a buffer for each chunk. Instead, get a [ReadableStreamBYOBReader] by calling `port.readable.getReader({ mode: "byob" })` and read data into a new `ArrayBuffer`. Note that the Web Serial API supports "Bring Your Own Buffer" readers in Chrome 106 or later.
```

Reply:
It is the stream allocating the buffer, not the developer. We're telling them that with a BYOB reader they can take over allocating buffers and reuse them as well. The message needs to be that it puts them in control.

Reply:
Reworded to clarify.

Reply:
How about,

```suggestion
You can take control of how memory is allocated when you read from the stream using a "Bring Your Own Buffer" reader. Call `port.readable.getReader({ mode: "byob" })` to get the [ReadableStreamBYOBReader] interface and provide your own `ArrayBuffer` when calling `read(). Note that the Web Serial API supports this feature in Chrome 106 or later.
```

Reply:
Sounds good.